create_static_html_files.py

```diff
@@ -299,20 +300,32 @@ def get_os_html(classifiers):
     return default_os_html
 
 
-def generate_home_html(home_pypi, home_github, home_other):
+def extract_github_info(url):
+    match = re.search(r"github\.com/([^/]+)/([^/]+)", url)
+    if match:
+        user, repo = match.group(1), match.group(2).rstrip(".git")
+        return user, repo
+    return None, None
+
+
+def generate_home_html(plugin_name, home_pypi, home_github, home_other):
     # Start with the PyPI link, which is always present
     html_content = f'''
    <div class="flex items-center" style="gap: 10px; ; align-items: center;"">
```

Comment:
Increase gap to 20px here and add suggested style to span below gives a quick and dirty style improvement.

**Full size**

![Screenshot 2025-06-25 at 11 10 00 AM](https://github.com/user-attachments/assets/53934f2b-2201-465a-b996-ea2016503e2b)

**Mobile size**

![Screenshot 2025-06-25 at 11 10 20 AM](https://github.com/user-attachments/assets/8591d23c-b0f9-4aed-9f65-17425bf66488)